tesseract_scene_graph/include/tesseract_scene_graph/kdl_parser.h

```diff
@@ -153,7 +162,8 @@ KDLTreeData parseSceneGraph(const SceneGraph& scene_graph);
  */
 KDLTreeData parseSceneGraph(const SceneGraph& scene_graph,
                             const std::vector<std::string>& joint_names,
-                            const std::unordered_map<std::string, double>& joint_values);
+                            const std::unordered_map<std::string, double>& joint_values,
+                            const tesseract_common::TransformMap& floating_joint_values);
```

Comment:
```suggestion
                            const tesseract_common::TransformMap& floating_joint_values = {});
```

Should this have a default value? I see several places (unit tests, etc.) where you pass in a default empty value because this function changed. Maybe easier to just provide it by default here

Reply:
I agree. I will amend the PR.